src/spikeinterface/core/template.py

```diff
@@ -30,9 +30,11 @@ class Templates:
         Array of unit IDs. If `None`, defaults to an array of increasing integers.
     probe: Probe, default: None
         A `probeinterface.Probe` object
+    is_scaled : bool, optional default: True
+        If True, it means that the templates are in uV, otherwise they are in raw ADC values.
     check_for_consistent_sparsity : bool, optional default: None
         When passing a sparsity_mask, this checks that the templates array is also sparse and that it matches the
-        structure fo the sparsity_masl.
+        structure fo the sparsity_mask. If False, this check is skipped.
```

Comment:
```suggestion
        structure of the sparsity_mask. If False, this check is skipped.
```

just so we don't forget this tiny typo :)

Reply:
Thanks Zach!